### DISCUSSION TITLE
Ce site n'utilise aucun cookie nécessitant votre consentement

### DISCUSSION THREAD

**Discussion by GRUXXKi** [AmauriC/tarteaucitron.js#1135] [Q&A] [answered]
Bonjour,

Bien qu'ayant installé trois services  facebookpixelId, intercom, gajsUa, multiplegoogletagmanager,  lorsque je clique pour ouvrir la bannière, j'ai toujours  "Ce site n'utilise aucun cookie nécessitant votre consentement".

Lorsque je log le contenu de la variable tarteaucitron, j'ai bien dans user les différents services de listé pourtant.

Est-ce que je rate potentiellement quelque-chose ?

J'ai installé la dernière version disponible des fichiers. 

Au passage j'ai un warning que je n'avais pas préalablement : Error with Permissions-Policy header: Unrecognized feature: 'ch-ua-form-factor'.

**Comment by AmauriC** [comment]
Bonjour :)

Possible d'avoir l'url du site que je vérifie?

Le warning n'a pas de lien avec tarteaucitron.

**Reply by GRUXXKi** [reply]
[LESITE]

**Reply by AmauriC** [reply, marked as answer]
Essayez de retirer le chargement que vous avez mis en dur du fichier tarteaucitron.fr.js
Ou si c'est volontaire parce que le script n'arrive pas à le charger, ajoutez aussi /tarteaucitron/tarteaucitron.services.js

**Reply by GRUXXKi** [reply]
Merci effectivement, ça règle le problème !

Et oui, pour une raison que je n'ai pas encore réussi à régler sous un un Symfony un peu ancien, impossible de les charger automatiquement.
